src/main/java/com/google/sps/app/servlets/NewTripServlet.java

```diff
@@ -94,6 +91,7 @@ public String validateInput(HttpServletRequest request) {
 
         String participantInput = StringEscapeUtils.escapeHtml4(request.getParameter(PARTICIPANTS_PARAM));
         List<String> participants = DataStoreHelper.splitUserList(participantInput);
+        if (participants.size() == 1 && participants.get(0).equals("")){return "";} // if there are no participants to add it skips this step
```

Comment:
Another option is for the splitUserList to never add an empty participant to the list. That way consumer of the list doesn't have to check. If you had unit tests, you could also add a test case for this in the splitUserList method.